packages/client/src/views/RegisterForm/duplicate/DuplicateFormTabs.tsx

```diff
@@ -647,8 +647,7 @@ export const DuplicateFormTabs = (props: IProps) => {
       )
 
       const duplicateRegData = {
-        status: eventData.history.find((data: History) => data.action === null)
-          .regStatus as RegStatus,
+        status: props.declaration.registrationStatus,
```

Comment:
This value needs to be read from `eventData` as this the other(duplicate) declaration. While the `props.declaration` contains the actual declaration we are comparing against

Reply:
@Siyasanga How about this one?